tests/integration/test_charm.py

```diff
@@ -117,3 +118,40 @@ async def test_tracing(ops_test: OpsTest):
     unit_relation_data = await _get_unit_relation_data(app, "tracing", side=PROVIDES)
 
     assert unit_relation_data
+
+
+async def test_integrate_blackbox(ops_test: OpsTest):
+    # @todo: upgrade to stable when blackbox charm with probes relation
+    # is promoted from edge.
+    await ops_test.model.deploy(
+        "blackbox-exporter-k8s", "blackbox", channel="latest/edge", trust=True
```

Comment:
How about using 'stable' ?

Reply:
because the charm parts in blackbox that handle the probes relation are only on edge at the moment, added a comment, with a TODO to upgrade once it is promoted.